cmd/buildah/main.go

```diff
@@ -168,14 +168,24 @@ func before(cmd *cobra.Command) error {
 	return nil
 }
 
-func after(cmd *cobra.Command) error {
+func shutdownStore(cmd *cobra.Command) error {
 	if needToShutdownStore {
 		store, err := getStore(cmd)
 		if err != nil {
 			return err
 		}
+		logrus.Debugf("shutting down the store")
+		needToShutdownStore = false
 		_, _ = store.Shutdown(false)
```

Comment:
Why are we ignoring the error? If it's intended, can you add a comment?

Reply:
Not my code...

Anyways, I did not want to risk making this a fatal error, so since it was ignored before, I will only treat it as an error.